pypeman/tests/test_channel.py

```diff
@@ -85,15 +85,16 @@ def setUp(self):
 
     def tearDown(self):
         super().tearDown()
-        self.clean_loop()
 
         for end in endpoints.all_endpoints:
             self.loop.run_until_complete(end.stop())
 
         # Stop all channels
         for chan in channels.all_channels:
-            if not chan.is_stopped:
+            if not chan.is_stopped():
+                print(f"stopping chan {chan.name}")
```

Comment:
print???

Reply:
Yup, loggers don't correctly works in test files, but this print was not useful so i remove it